llvm/include/llvm/Transforms/Vectorize/SandboxVectorizer/Interval.h

```diff
@@ -176,6 +176,26 @@ template <typename T> class Interval {
       Result.emplace_back(Intersection.To->getNextNode(), To);
     return Result;
   }
+  /// A difference that asserts that the result is a single interval.
```

Comment:
\Returns a

Reply:
this only asserts in asserts-enabled builds, perhaps drop the "that asserts"?

Reply:
I will change it to:
`/// \Returns the interval difference `this - Other`. This will crash in the Debug build if the result is not a single interval.`
Wdyt ?